packages/ui/src/council/types/Vote.ts

```diff
@@ -32,7 +33,7 @@ export type PastElectionVote = BaseVote
 
 export const asVote = (fields: CastVoteFieldsFragment): Vote => ({
   id: fields.id,
-  createdAt: fields.createdAt,
+  createdAtBlock: fields.castEvent ? asBlock(fields.castEvent[0]) : randomBlock(),
```

Comment:
I think this is more typing mistake, so for now just updated types. I assume that we always have these events on past votes/elections.

Reply:
Hopefully `any[]` is enough for now.

Reply:
Please try to not use `any` :cry:  The `any` will just disable any type check whatsoever and we don't want that.

I see that the vote array is optional, so an unknown block might be better. It might render to an empty component.